.github/scripts/check_integration_test_label.py

```
@@ -0,0 +1,188 @@
+#!/usr/bin/env python3
```

Pylint (4.0.2)

[error] Unrecognized option found: optimize-ast, files-output, variable-name-hint, class-attribute-name-hint, argument-name-hint, module-name-hint, const-name-hint, inlinevar-name-hint, method-name-hint, function-name-hint, attr-name-hint, class-name-hint, no-space-check

(E0015)